### ISSUE TITLE
Loss is not decreasing and weight values are not updating for train.py --mod mat

### ISSUE THREAD

**Issue by Raj-08** [takiyu/portrait_matting#7]
I have made sure i am sending in the right data , with the right values. 
Alpha matte - [0.255]
All of the weight values are zero.

**Comment by leaf918** [comment]
does it works?

**Comment by kkkmax** [comment]
When I use the command :
        $python scripts/train.py -mode  mat , 
my loss curve does not drop. Have you solved this problem?

![5](https://user-images.githubusercontent.com/42377340/50737507-2831ce80-1205-11e9-81ca-8acc33246c8b.png)



> 我已确保使用正确的值发送正确的数据。
> Alpha遮罩 - [0.255] 
> 所有重量值均为零。

When I use the command :
        $python scripts/train.py -mode  mat , 
my loss curve does not drop. Have you solved this problem?

**Comment by pourfard** [comment]
Leaving learning rate (lr) parameter with it's default value may slow down training. Setting "--lr 0.001" makes training much faster and probably you can see decreasing loss in early epochs.
